examples/daymet-hi-annual.json

```diff
@@ -0,0 +1,186 @@
+{
+    "type": "Collection",
+    "stac_version": "1.0.0-rc.3",
+    "stac_extensions": [
+        "https://stac-extensions.github.io/datacube/v1.0.0/schema.json"
+    ],
+    "id": "daymet-hi-annual",
+    "title": "Daymet Hawaii Annual",
+    "description": "Daymet is a data product derived from a collection of algorithms and computer software designed to interpolate and extrapolate from daily meteorological observations to produce gridded estimates of daily weather parameters. Weather parameters generated include daily surfaces of minimum and maximum temperature, precipitation, vapor pressure, radiation, snow water equivalent, and day length produced on a 1 km x 1 km gridded surface. The motivation for producing Daymet is to provide measurements of near-surface meteorological conditions where no instrumentation exists. Having estimates of these surfaces is critical to understanding many processes in the terrestrial biogeochemical system.",
+    "license": "proprietary",
+    "extent": {
+        "spatial": {
+            "bbox": [
+                [
+                    -160.3056,
+                    17.9539,
+                    -154.772,
+                    23.5186
+                ]
+            ]
+        },
+        "temporal": {
+            "interval": [
+                [
+                    "1980-01-01T12:00:00Z",
+                    "2019-12-31T12:00:00Z"
+                ]
+            ]
+        }
+    },
+    "links": [],
+    "assets": {
+        "data": {
+            "href": "az://daymet-zarr/annual/hi.zarr",
+            "title": "Zarr store",
+            "type": "application/fsspec/zarr",
+            "description": "Root URL of the Zarr store",
+            "roles": [
+                "data"
+            ]
```

Comment:
This is likely wrong, but the intent is to provide a URL that can be loaded with xarray / zarr, which represents the entire collection / dataset.

```python

store = fsspec.get_mapper('az://daymet-zarr/daily/hi.zarr', account_name="daymeteuwest")
ds = xr.open_zarr(store, consolidated=True)
```

That said, I have *no* idea what the right media type would be for that.

Reply:
If there's no media type and we can't agree on one with zarr folks, we may just not provide a type for now and instead specify a zarr role, e.g. `roles: ["data", "zarr-collection"]` or however this should be called...

By the way, what is the az:// protocol?

Reply:
> By the way, what is the az:// protocol?

It's specific to fsspec / adlfs, unfortunately: https://github.com/dask/adlfs/. No clients outside of that ecosystem would (correctly) interpret it.